src/tests/ftest/util/run_utils.py

```diff
@@ -352,9 +352,9 @@ def run_local(log, command, capture_output=True, timeout=None, check=False, verb
 
     except subprocess.CalledProcessError as error:
         # Raised if command yields a non-zero return status with check=True
-        log.debug(str(error))
-        log.debug("  output: %s", error.output)
-        log.debug("  stderr: %s", error.stderr)
+        log.info(str(error))
+        log.info("  output: %s", error.output)
+        log.info("  stderr: %s", error.stderr)
```

Comment:
Why?

Reply:
I wondered where this commit had gone, I meant to make it to a different PR.  I've re-pushed this PR with Test-tags set to get debugging information for a specific test failure we're seeing.

Reply:
But why info instead of debug? debug should still show up in the log, and everything else in this function is debug

Reply:
If launch.py crashes then the log isn't saved for some reason.  It's complicated.